Agents/Xamarin.Interactive/CodeAnalysis/EvaluationAssemblyContext.cs

```diff
@@ -16,14 +16,33 @@ namespace Xamarin.Interactive.CodeAnalysis
 {
     sealed class EvaluationAssemblyContext : IDisposable
     {
+        sealed class AssemblyNameInsensitiveNameOnlyComparer : IEqualityComparer<AssemblyName>
```

Comment:
Maybe add a comment that includes info about the bug fixed here, and the bug linked in the old comment below?